src/modules/mount/main.py

```diff
@@ -149,6 +149,21 @@ def get_btrfs_subvolumes(partitions):
 
     return btrfs_subvolumes
 
+def enable_swap_partition():
+    swap_is_exist=subprocess.check_output(["swapon","-s"], universal_newlines=True)
+    if ( swap_is_exist == "" ):
+        all_partitions=subprocess.check_output(["lsblk", "-l", "-f", "-n", "-p"], universal_newlines=True).split("\n")
```

Comment:
The list of partitions is already available in `partitions` if you read above a little in the code.  That should be used instead of a call to `lsblk`.

Using this same data structure, you should also ensure the partition is "claimed".  Otherwise we could overwrite data in a swap partition the user had not selected to use.  There are examples of using this in the fstab module if you want a reference.